src/Components/Search/NewSearch/Mobile/Overlay.tsx

```diff
@@ -44,11 +44,12 @@ export const Overlay: FC<OverlayProps> = ({ viewer, relay, onClose }) => {
     tracking.trackEvent({
       action_type:
         options.length > 0
-          ? DeprecatedSchema.ActionType.SearchedAutosuggestWithResults
-          : DeprecatedSchema.ActionType.SearchedAutosuggestWithoutResults,
+          ? ActionType.selectedSearchSuggestionQuickNavigationItem
```

Comment:
**suggestion**: I think this "QuickNavigationItem" refers to the (Artworks) or (Auction Results) links in Artist results and not a search result/suggestion itself. We might need to stick with these deprecated action types as I think they more accurately reflect what this event is meant to track.